.pre-commit-config.yaml

```diff
@@ -14,21 +14,21 @@ repos:
       - id: check-case-conflict
       - id: detect-private-key
 
-  - repo: https://github.com/lyz-code/yamlfix/
-    rev: 1.16.0
-    hooks:
-      - id: yamlfix
-        args: [-c, .yamlfix.toml]
+  # - repo: https://github.com/lyz-code/yamlfix/
```

Comment:
I needed to disable yamlfix for now because it was generating errors related to the tool itself